default/python/common/charting/charts.py

```diff
@@ -17,6 +17,7 @@
 # Change dataDir to '<user>/AppData/Roaming/'
 
 
+from __future__ import print_function
```

Comment:
Does anyone use this code?  It was initially written by @Dilvish-fo.

Reply:
It's a script draws graphs based on logs (it don't work with non-ascii). I've tried it and it works for me with en_US locale.